go.mod

```diff
@@ -10,6 +10,7 @@ require (
 	github.com/aws/aws-sdk-go v1.38.41
 	github.com/blang/semver/v4 v4.0.0
 	github.com/davecgh/go-spew v1.1.1
+	github.com/davegardnerisme/deephash v0.0.0-20210406090112-6d072427d830
```

Comment:
maybe the use what we do elsewhere in the code, https://github.com/openshift/hive/blob/57147b24cb33b05de91c8070b459b66c24556e06/pkg/operator/hive/controllersconfig.go#L86-L91

Reply:
I'll look into that. Is `%v` guaranteed to produce the same thing for (different instances of) identical objects? I know some languages [don't/didn't guarantee map ordering](https://docs.python.org/2.7/library/stdtypes.html#dict.items) for example.

Reply:
So yeah, I can't prove that `%v` breaks with map key order, but I can prove it doesn't play nice with pointers, whereas `deephash` does The Right Thing™: https://play.golang.org/p/1SQUEeJfOND